projects/1.16/assets/tetra/tetra/lang/zh_cn.json

```
@@ -2031,13 +2034,13 @@
   "tetra.advancements.spelunking.hammer_unjam.title": "获得维护",
   "tetra.advancements.spelunking.hammer_unjam.description": "用一个至少四级的手持锤子敲击故障的锤头，将锻造锤恢复到工作状态",

```

```suggestion
  "tetra.advancements.spelunking.hammer_unjam.description": "用一个至少4级的手持锤子敲击故障的锤头，将锻造锤恢复到工作状态",

```
（之前都是阿拉伯数字，这里用汉字有些突兀）